### ISSUE TITLE
Option to hide h1

### ISSUE THREAD

**Issue by emufan** [alexarch21/history-explorer-card#24]
Currently, when the title has a space, no title is shown, but still the line (gap).

Would be great, if the whole line is not displayed or at least the user can choose, e.g. _ show blank and __ hides the row or whatever.

Currently, I’m using.

```
    card_mod:
      style: |
        h1 {
          display: none;
        }
```

**Comment by Bellavista** [comment]
Hi emufan
I don't know if this solves your request
if you use header: ' ' 'space character' in the header name you have this result
![image](https://user-images.githubusercontent.com/46230198/159177303-cd16b34f-5c76-455a-96fc-efc00f8d3ba1.png)
but if you use header: [] square brackets
![image](https://user-images.githubusercontent.com/46230198/159177409-6aa56ae2-bab5-4074-a0af-e6dcdecb9345.png)

**Comment by emufan** [comment]
Yes. Working. Thank you! Didn't know it or saw this documented before. Will add a PR for the documentation.

**Comment by alexarch21** [comment]
I'm reopening this. The [] is an unintended side effect of the code that is very fragile and could easily break in future releases. You can use it in the mean time to remove the header, but I'm going to add proper support for this option.

Some more info on the technical details in @emufan 's PR.

**Comment by alexarch21** [comment]
Fixed in v1.0.19. To remove the header completely, use `header: hide` in the config.
